src/stash/tests/stash.rs

```diff
@@ -17,24 +17,24 @@ use mz_stash::{
     TypedCollection,
 };
 
-#[test]
-fn test_stash_sqlite() -> Result<(), anyhow::Error> {
+#[tokio::test]
+async fn test_stash_sqlite() -> Result<(), anyhow::Error> {
     {
         let file = NamedTempFile::new()?;
         let mut conn = Sqlite::open(file.path())?;
-        test_stash(&mut conn)?;
+        test_stash(&mut conn).await?;
     }
     {
         let file = NamedTempFile::new()?;
         let mut conn = Sqlite::open(file.path())?;
-        test_append(&mut conn)?;
+        test_append(&mut conn).await?;
     }
     Ok(())
 }
 
-#[test]
-fn test_stash_postgres() -> Result<(), anyhow::Error> {
-    use postgres::{Client, NoTls};
+#[tokio::test]
```

Comment:
note that the default for `tokio::test` is the single-threaded runtime, if you want some sort of exercise on multiple threads, you can configure it as such